paddlets/analysis/frequency_domain.py

```diff
@@ -225,16 +225,16 @@ class STFT(Analyzer):
 
     def __init__(
         self,
-        fs: float = 1.0,
-        window: Union[str, Tuple[str], List[str]] = 'hann',
-        nperseg: int = 256,
-        noverlap: Union[None, int] = None,
-        nfft: Union[None, int] = None,
-        detrend: Union[str, bool] = False,
-        return_onesided: bool = True,
-        boundary: Union[str, None] = 'zeros',
-        padded: bool = True,
-        axis: int = -1,
+        fs: Optional[float] = 1.0,
+        window: Optional[Union[str, Tuple[str], List[str]]] = 'hann',
```

Comment:
should Not be Optional, refers to https://docs.python.org/3/library/typing.html#typing.Optional